.travis.yml

```diff
@@ -10,17 +10,17 @@ jobs:
       env: CMAKE_BUILD_TYPE=Debug DIST=bionic CTEST_OUTPUT_ON_FAILURE=1
     - dist: bionic
       env: CMAKE_BUILD_TYPE=Release DIST=bionic CTEST_OUTPUT_ON_FAILURE=1
-    - dist: xenial
-      env: CMAKE_BUILD_TYPE=Debug DIST=xenial CTEST_OUTPUT_ON_FAILURE=1 CMAKE_CXX_STANDARD=11
-    - dist: xenial
-      env: CMAKE_BUILD_TYPE=Release DIST=xenial CTEST_OUTPUT_ON_FAILURE=1 CMAKE_CXX_STANDARD=11
+    - dist: focal
```

Comment:
The python packages installed in 20.04 are different from those in `18.04`. The focal tests are failing.
I would suggest to:
1) either do the full CI integration here, or,
2) revert this comment and let @nim65s handle it in a different PR?

Reply:
I suggested to move from xenial to focal, but I didn't look at all the implications. Here, I guess this is only a matter of `s/python-scipy/python3-scipy`.

Reply:
For focal, I am using the following libraries:
https://github.com/loco-3d/crocoddyl/blob/9f87028f8376673fcdf49e8f252d6607ecade5c0/.travis/install-prerequisites.sh#L10

So, do I need to replace `python-scipy` by `python3-scipy`? Is there something else to be done (e.g., adding `which pyhton3`)?

Reply:
I have solved the issue with Travis. Waiting that all jobs pass to merge this PR.

Thanks guys :)